site/e2e/tests/auditLogs.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { expect, test } from "@playwright/test";
+import {
+  createTemplate,
+  createWorkspace,
+  requiresEnterpriseLicense,
+} from "../helpers";
+import { beforeCoderTest } from "../hooks";
+
+test.beforeEach(async ({ page }) => await beforeCoderTest(page));
+
+test("inspecting and filtering audit logs", async ({ page }) => {
+  requiresEnterpriseLicense();
+
+  const userName = "admin";
+  // Do some stuff that should show up in the audit logs
+  const templateName = await createTemplate(page);
+  const workspaceName = await createWorkspace(page, templateName);
+
+  // Go to the audit history
+  await page.goto("/audit");
+
+  // Make sure those things we did all actually show up
+  await expect(page.getByText(`${userName} logged in`)).toBeVisible();
+  await expect(
+    page.getByText(`${userName} created template ${templateName}`),
+  ).toBeVisible();
+  await expect(
+    page.getByText(`${userName} created workspace ${workspaceName}`),
+  ).toBeVisible();
+  await expect(
+    page.getByText(`${userName} started workspace ${workspaceName}`),
+  ).toBeVisible();
+
+  // Make sure we can inspect the details of the log item
+  const createdWorkspace = page.locator(".MuiTableRow-root", {
+    hasText: `${userName} created workspace ${workspaceName}`,
+  });
+  await createdWorkspace.getByLabel("open-dropdown").click();
```

Comment:
Is this a label that we're visibly showing in the UI? The hyphen makes it look like an implementation detail

Reply:
It's an `aria-label`. Which probably still shouldn't have a hyphen, but it is what it is. 😅